ovh/error_test.go

```diff
@@ -8,17 +8,27 @@ import (
 )
 
 func TestErrorString(t *testing.T) {
-	err := &APIError{
+	err := APIError{
 		Code:    http.StatusBadRequest,
-		Message: "Bad request",
+		Message: "An input error occurred",
 	}
-	td.CmpString(t, err, `HTTP Error 400: "Bad request"`)
+	expected := `OVHcloud API error (status code 400): "An input error occurred"`
+	td.Cmp(t, err.Error(), expected)
+	td.Cmp(t, err.String(), expected)
 
-	err = &APIError{
+	err = APIError{
 		Code:    http.StatusConflict,
 		Message: `the cart id "foobar" already exists`,
 		Class:   "CartAlreadyExists",
 		QueryID: "EU.ext-99.foobar",
 	}
-	td.CmpString(t, err, `HTTP Error 409: CartAlreadyExists: "the cart id \"foobar\" already exists" (X-OVH-Query-Id: EU.ext-99.foobar)`)
+	expected = `OVHcloud API error (status code 409): CartAlreadyExists: "the cart id \"foobar\" already exists" (X-OVH-Query-Id: EU.ext-99.foobar)`
+	td.Cmp(t, err.Error(), expected)
+	td.Cmp(t, err.String(), expected)
+
+	err.Class = ""
+	expected = `OVHcloud API error (status code 409): "the cart id \"foobar\" already exists" (X-OVH-Query-Id: EU.ext-99.foobar)`
+	td.Cmp(t, err.Error(), expected)
+	td.Cmp(t, err.String(), expected)
+
```

Comment:
```suggestion
```

Reply:
My editor run linter that add a final line to every file, I guess this one is non-blocking and I will not apply.